Prajna.sln

```diff
@@ -572,6 +574,22 @@ Global
 		{E95D819A-B8E3-4F94-B7EE-300E31AB0181}.Release|Win32.Build.0 = Release|Win32
 		{E95D819A-B8E3-4F94-B7EE-300E31AB0181}.Release|x64.ActiveCfg = Release|x64
 		{E95D819A-B8E3-4F94-B7EE-300E31AB0181}.Release|x64.Build.0 = Release|x64
+		{6797EE77-90E4-4719-BC81-F3FA2840CA9A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
To be consistent with all other projects within this sln, it should only contain x64.